specification/azsadmin/resource-manager/commerce/readme.azsautogen.md

```diff
@@ -0,0 +1,8 @@
+# AutoRest v3 generators
```

Comment:
May I know what is this file used for?

Reply:
This is just a config addition for generating the powershell cmdlets using autorest powershell generator.

Reply:
Got it